static-site/content/community-datasets.yaml

```diff
@@ -216,3 +216,7 @@ data:
     maintainers: Albert Rock A. Gangbadja (TIDRC, UAC, Abomey-Calavi, Benin)
     date: '2023-11-22'
     title: Genomic epidemiology of SARS-CoV-2 in CAMES countries
+  - url: https://nextstrain.org/community/joewinnz/WheatBlast
```

Comment:
Thank you for sharing your community dataset @joewinnz! 

I just wanted to note that the Auspice JSON file name must start with the repository name. There is a mismatch in capitalization in your Auspice JSON file name [`wheatblast.json`](https://github.com/joewinnz/WheatBlast/blob/253df10d39e1bf4850bad173e407ac27d900152c/auspice/wheatblast.json) and your repository name `WheatBlast`. As a result, your community build is not currently viewable at https://nextstrain.org/community/joewinnz/WheatBlast/. 

If you update the file name to `WheatBlast.json` then the dataset will be viewable at the URL.

Reply:
The file was renamed recently https://github.com/joewinnz/WheatBlast/commit/c01c5e846a416310910e41cdf91aeb853c9d96eb and now https://nextstrain.org/community/joewinnz/WheatBlast/ works. I'll merge this PR to include the dataset in the community listing.